src/index.ts

```diff
@@ -24,6 +24,12 @@ export interface CheckoutManifest {
   }
 }
 
+/** Represents information sent when the Checkout process succeeds. */
+export interface CheckoutPaymentInfo {
+  /** Whether the payment was completed (for synchronous payment methods) or not (for asynchronous ones). */
```

Comment:
On asynchronous it is not completed?

Reply:
Then I'd say to change the `...or not` part and change to `or requested (for async...)` to be more specific .

Reply:
Will do.